react-app/package.json

```
@@ -23,6 +23,8 @@
     "@mui/icons-material": "5.15.6",
     "@mui/material": "5.15.6",
     "@mui/x-data-grid": "6.19.2",
+    "@mui/x-date-pickers": "^6.19.5",
+    "dayjs": "^1.11.10",
```

```suggestion
    "@mui/x-date-pickers": "6.19.5",
    "dayjs": "1.11.10",
```
Removing '^'s so that the ticket creation script wont flag false positives.